Jenkinsfile

```diff
@@ -27,7 +25,7 @@ def getVariables(repoName) {
 }
 
 def buildTestImage(name, suffix) {
-  sh 'docker image prune -f'
+  sh 'docker image prune -f || echo could not prune images'
```

Comment:
Should we still have this here? Now we've moved to a nightly task to prune images, this just risks stepping on other jobs that might be running at the same time.